QoLPack/jars/src/data/plugins/qolp_modPlugin.java

```diff
@@ -12,17 +12,84 @@
     public static final String ID = "qolp_modPlugin";
     public static final String SETTINGS_PATH = "QoLPack.ini";
 
+    private JSONObject settings;
+
+    @Override
+    public void afterGameSave() {
+        if (hasQol("ScavengeAsYouFly")) {
+            restoreAutoScavenge();
+        }
+    }
+
+    @Override
+    public void beforeGameSave() {
+        if (hasQol("ScavengeAsYouFly")) {
+            removeAutoScavenge();
```

Comment:
This is critical to being a utility mod. We need to remove this ability from the save. We readd it back after save, and preserve the state of it via memory key.